TAO/TAO_IDL/be_include/be_helper.h

```diff
@@ -167,6 +172,22 @@ class TAO_OutStream
   int gen_endif (void);
 
   // =overloaded operators
+#if defined (ACE_HAS_CPP11)
+  // Avoid duplication for the underlying type of size_t
+  template <typename Dummy = TAO_OutStream &>
+  typename std::enable_if<std::is_same<Dummy, TAO_OutStream &>::value &&
+                          !std::is_same<ACE_CDR::ULongLong, size_t>::value &&
+                          !std::is_same<ACE_CDR::ULong, size_t>::value,
+                          TAO_OutStream &>::type
+  operator << (const size_t num)
+  {
+    ACE_OS::fprintf (this->fp_,
+                     ACE_TEXT_ALWAYS_CHAR (ACE_SIZE_T_FORMAT_SPECIFIER),
```

Comment:
```suggestion
                     ACE_SIZE_T_FORMAT_SPECIFIER_ASCII,
```
Avoids going through an encoding conversion at runtime (if wchar build).

Reply:
The overloaded functions for ACE_CDR::LongLong and ACE_CDR::ULongLong are currently doing this conversion. Do you think we should change those too?

Reply:
Yes, I don't think this combination `ACE_TEXT_ALWAYS_CHAR (ACE_SIZE_T_FORMAT_SPECIFIER)` really makes sense when there are at least 2 better alternatives:
- `ACE_SIZE_T_FORMAT_SPECIFIER_ASCII` which has type `char*` on all builds
- Just  `ACE_SIZE_T_FORMAT_SPECIFIER` which has type `wchar_t*` on wchar-enabled builds, so this calls `ACE_OS::fprintf(FILE*, const wchar_t*, ...)`

Reply:
Those are fixed.